cwolfmap/cwolfmap.c

```diff
@@ -191,13 +191,19 @@ static int LoadMapHead(CWolfMap *map, const char *path)
 	// Read as many maps as we can; some versions of the game (SOD MP) truncate
 	// the headers
 	(void)!fread((void *)&map->mapHead, 1, size, f);
+	map->mapHead.magic = htole16(map->mapHead.magic);
```

Comment:
```suggestion
	map->mapHead.magic = letoh16(map->mapHead.magic);
```
I think this should be right?